src/routes/things.js

```diff
@@ -47,8 +56,74 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((models) => {
+    if (req.params && req.params[0]) {
+      models.Things.findById(req.params[0]).then(thing => {
+        res.status(200).send(response.generate(thing));
+      });
+    } else {
+      models.Things.findAll().then(things => {
+        res.status(200).send(response.generate(things));
+      });
+    }
+  })
+  .catch(() => {
+    ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR);
+  });
+});
+
+router.post('/', (req, res) => {
+  db().then((models) => {
```

Comment:
This is ok for now, but add a comment `// XXX #IssueNumber [Things router] Implement associations to other models and integrity check` and create an issue for it, please.